### PR TITLE
Directory/trunk

### PR DESCRIPTION
**Pull request by bungaepyo**

### CODE REVIEW

File: src/components/directory/StateLinks.tsx
Comment (awang8) [inline src/components/directory/StateLinks.tsx]: Is there a way we can add this to the .env file or something instead of hard coding it here ?

File: src/templates/robots.ts
Comment (awang8) [inline src/templates/robots.ts:21]: What is this file for ?

Comment (bungaepyo) [reply src/templates/robots.ts:21]: This is for the robots.txt page [here](https://directoryqa-seemly--realistic--impala-pgsdemo-com.preview.pagescdn.com/robots.txt), which has rules set out for web crawlers. This was included in project phase 1 for SEO purposes.

File: src/components/directory/StateLinks.tsx
Comment (bungaepyo) [reply src/components/directory/StateLinks.tsx]: Yes, we can add this Streams API key to both site settings and .env file.

Comment (bungaepyo) [reply src/components/directory/StateLinks.tsx]: https://github.com/ttverifiedpro/itvp-pro-website/pull/8

**Review by bheus — APPROVED** [review]
Approving, let's hold off on merging until we get our Definition of Done (DoD) approval

**Review by awang8 — APPROVED** [review]

**Comment by bungaepyo** [comment]
@cblair28 @moose-byte @awang8 I've added `directory/trunk` as a tracked branch in the `Pro pages (Prod)` site so we can preview the content on it before merging. Hope this helps.

Staging deploy of branch: https://www.yext.com/s/4075371/yextsites/83089/branch/102298/deploys/recent
Staging page: https://directorytrunk-pros-turbotax-intuit-com-pagescdn-com.preview.pagescdn.com/local-tax-experts

**Pull request merged**
